src/modules/Builder/features/SaveAndPublish/SaveAndPublish.utils.ts

```diff
@@ -117,7 +117,11 @@ export const remapSubscaleSettings = (activity: ActivityFormValues) => {
   } as NonNullable<ActivityFormValues['subscaleSetting']>;
 };
 
-const getConditions = (items: ItemFormValues[], conditions?: (SectionCondition | Condition)[]) =>
+const getConditions = (
+  items: ItemFormValues[],
+  conditions?: (SectionCondition | Condition)[],
+  score?: ScoreReport,
+) =>
```

Comment:
could you please use an object for parameters instead of sequence of them?

Reply:
done